recipes/rapidjson/bld.bat

```diff
@@ -0,0 +1,4 @@
+set RAPIDJSON_DIR=%SRC_DIR%
+
+REM Copy headers
+xcopy /S %RAPIDJSON_DIR%\include %LIBRARY_INC%
```

Comment:
This is a bit outside my expertise -- can you propose a new recipe (feel free to start from this one if you'd like) with the Windows fixes? I am happy to be on the maintainer list